### PR TITLE
Update wtag SF

### PR DESCRIPTION
**Pull request by siqiyyyy**
According to update here https://www.dropbox.com/s/z1rnuaxrekmrvlu/DeepAK8_Top_W_SFs_V2_Training_FINAL_w_Tables.pdf?dl=0

### CODE REVIEW

**Comment by siqiyyyy** [comment]
Let me also update the WPs before merging

File: bucoffea/config/monojet.yaml [hunk 1]
```diff
@@ -171,8 +171,8 @@ default:
         - 79123963
         - 51393664
     trigger_study: False
-    monov: False
```
Comment (AndreasAlbert) [inline bucoffea/config/monojet.yaml]: I know its kind of annoying, but could we switch this back?

File: bucoffea/config/monojet.yaml [hunk 2]
```diff
@@ -465,12 +465,11 @@ era2018:
         loose : 0.1241
         medium: 0.4184
         tight: 0.7527
-  wtag: # https://indico.cern.ch/event/840827/contributions/3527925/attachments/1895214/3126510/DeepAK8_Top_W_SFs_2017_JMAR_PK.pdf
```
Comment (AndreasAlbert) [inline bucoffea/config/monojet.yaml:468]: For future reference, could you add the slides here?

File: bucoffea/config/monojet.yaml [hunk 3]
```diff
@@ -526,7 +525,7 @@ era2018:
       histogram: NUM_LooseRelIso_DEN_LooseID_pt_abseta_syst
       file: data/sf/muon/2018_RunABCD_SF_ISO.root
 
-    wtag_loose: # https://indico.cern.ch/event/840827/contributions/3527925/attachments/1895214/3126510/DeepAK8_Top_W_SFs_2017_JMAR_PK.pdf
```
Comment (AndreasAlbert) [inline bucoffea/config/monojet.yaml:529]: same

**Review by AndreasAlbert — CHANGES_REQUESTED** [review]

**Comment by siqiyyyy** [comment]
Added those changes

**Pull request merged**